randovania/games/prime1/pickup_database/pickup-database.json

```diff
@@ -177,7 +181,8 @@
             "offworld_models": {
                 "am2r": "sItemMissile",
                 "dread": "powerup_missile",
-                "prime2": "MissileExpansionPrime1"
+                "prime2": "MissileExpansionPrime1",
+                "samus_returns": "powerup_missilelauncher"
```

Comment:
this is supposed to be an expansion until prime gets properly working required mains

Reply:
Why is that?

Reply:
cant find the exact reasoning, but was something along the lines of being less confusing.

Reply:
I dont see how it's more confusing. Having it be a tank would be so much worse since there's so many of them

Reply:
Actually probably doesn't matter since I forgot mains aren't a thing anyway